tests/parse.spec.js

```diff
@@ -232,4 +232,54 @@ describe('Parse', function() {
     ]);
   });
 
+  it('should handle unknown escape sequences inside a value', function() {
+    m.chai.expect(parse([
+      'device: /dev/disk1',
+      'description: "\\]=-01`234567890=-098e"'
+    ].join('\n'))).to.deep.equal([
+      {
+        device: '/dev/disk1',
+        description: ']=-01`234567890=-098e'
+      }
+    ]);
+  });
+
+  it('should delete known escape sequences', function() {
```

Comment:
Yeah, that's a good point. My changes in #156 convert the Linux mount-specific escape sequences to 'regular ASCII escapes', and then your changes here remove those regular ASCII escape sequences. That sounds like the correct separation of logic.

Although now I think about it, perhaps the changes you're making in this PR are Etcher-specific, and there might be other users of `drivelist` who _want_ these escape sequences to be preserved? Perhaps there should be an optional parameter that a user could supply to drivelist, specifying whether they want these escape sequences preserved or not?